src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -332,52 +426,228 @@
      [react/scroll-view
       [view]]]))
 
+(defonce scroll (reagent/atom nil))
+(def initial-on-show-done? (volatile! false))
+(def hidden (reagent/atom 0))
+(def shown (reagent/atom 100))
+(def header-profile-visible? (reagent/atom false))
+
+(defonce viewability-config-callback-pairs
```

Comment:
could you elaborate why is it so complex? don't we want just to show name in the toolbar?

Reply:
It is so complex because in https://framer.cloud/AvjzY/ take note of the shadow under the normal size toolbar; which also animates(fades-in/out) at the speed of scroll;

additionally(not in framer.cloud prototype), I added the "feature" 😅 of the thin-separator-line below Big profile(big toolbar) header to fade out also